srcpkgs/vpnd/files/vpnd/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+install -d -m0700 /var/run/vpnd
```

Comment:
What is the purpose of this directory? I see no mention of it in `/usr/bin/vpnd` or `/etc/vpnd.conf`.

Reply:
Oh my goodness! I am sorry. This was a leftover from an other template. I'll remove it ASAP. Thanks

Reply:
Fixed.